qui/decorators.py

```diff
@@ -223,11 +223,11 @@ def icon(self) -> Gtk.Image:
         return icon_img
 
     def netvm(self) -> Gtk.Label:
-        netvm = self.vm.netvm
+        netvm = getattr(self.vm, 'netvm', None)
```

Comment:
This should distinguish between "no netvm set" and "unable to read netvm" - otherwise user may think some qube is isolated from the network, while it isn't.